src/Rendering/babylon.outlineRenderer.ts

```diff
@@ -24,6 +24,13 @@
             var material = subMesh.getMaterial();
 
             engine.enableEffect(this._effect);
+
+            // Logarithmic depth
+            if(material instanceof StandardMaterial && material.useLogarithmicDepth)

```

Comment:
no need to check if instanceOf..Just test if useLogarithmicDepth is true (this will work for PBR as well )

Reply:
typescript compiler won't like it?

Reply:
Cast it to <any> :)

Reply:
Do you mean this:
```
var logMat = <PBRBaseMaterial> material || <StandardMaterial> material;
if(logMat && logMat.useLogarithmicDepth)
//...
```
?

Reply:
Nope I really mean <any>

Reply:
Sorry < any >

Reply:
Yes:)